test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestampFn } from '../helpers/utils'
+
+describe('JBProjects::challengeHandle(...)', function () {
+
+  let jbOperatorStoreFactory;
+  let jbOperatorStore;
+
+  let jbProjectsFactory;
+  let jbProjectsStore;
+
+  let deployer;
+  let projectOwner;
+  let addrs;
+
+  let projectHandle = "PROJECT_1";
+  let projectHandleNotTaken = "PROJECT_2"
+  let projectHandleNotTaken2 = "PROJECT_3"
+  let metadataCid = "";
+
+  beforeEach(async function () {
+    [deployer, projectOwner, ...addrs] = await ethers.getSigners();
+
+    jbOperatorStoreFactory = await ethers.getContractFactory('JBOperatorStore');
+    jbOperatorStore = await jbOperatorStoreFactory.deploy();
+
+    jbProjectsFactory = await ethers.getContractFactory('JBProjects');
+    jbProjectsStore = await jbProjectsFactory.deploy(jbOperatorStore.address);
+  });
```

Comment:
We've adopted a new pattern for test setup. Refer to the JBDirectory tests for more details – trying to do away with cross-test stateful variables whenever possible. Replace in this file and all others.